src/helm/env.d/dev/values.desk.yaml.gotmpl

```diff
@@ -47,7 +47,10 @@ backend:
         name: backend
         key: OIDC_RP_CLIENT_SECRET
     OIDC_RP_SIGN_ALGO: RS256
-    OIDC_RP_SCOPES: "openid email siret"
+    OIDC_RP_SCOPES: "openid email siret given_name usual_name"
+    USER_OIDC_FIRST_NAME_FIELD: "given_name"
+    USER_OIDC_LAST_NAME_FIELD: "usual_name"
```

Comment:
Please don't :) Those settings will be added in a future PR

Reply:
I can't crack WHY but it stops working when I remove it. And of course I restarted tilt, and tried on and off and on again and I definitely need them. 

I use the `tilt-up` stack, no keycloak

Reply:
I know it doesn't make any sense and I can't crack WHY but name retrieval stops working when I remove it. Of course I restart tilt eveytime I try, and tried on and off and on again and it definitely seem to need them. 
.
I use the `tilt-up` stack by the way, no keycloak